ui/components-react/src/components-react/newEditors/editors/text/UseTextEditorProps.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { EditorProps, RequiredProps } from "../../Types.js";
+import type { TextValue, Value } from "../../values/Values.js";
+import { isTextValue } from "../../values/Values.js";
+
+/**
+ * Hooks that converts generic `EditorProps` into editor props with text value. If value is not text, it will be converted into empty text value.
```

Comment:
> If value is not text, it will be converted into empty text value.

Maybe we should attempt to convert the value to its text representation instead?

Reply:
Removed conversion.